ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-commands.sh

```
@@ -21,24 +21,24 @@ extract_version() {
-  echo "### Copying metallb directory"
-  scp "${SSHOPTS[@]}" -r "${METALLB_SRC_DIR}" "root@${IP}:/root/dev-scripts/"
-else
-  echo "### Cloning metallb"
-  ssh "${SSHOPTS[@]}" "root@${IP}" "cd /root/dev-scripts/ && rm -rf metallb/ && git clone ${METALLB_REPO} && cd metallb/ && git checkout ${METALLB_BRANCH}"
-fi
+#if [ -d "${METALLB_SRC_DIR}" ]; then
+#  echo "### Copying metallb directory"
+#  scp "${SSHOPTS[@]}" -r "${METALLB_SRC_DIR}" "root@${IP}:/root/dev-scripts/"
+#else
+echo "### Cloning metallb"
+ssh "${SSHOPTS[@]}" "root@${IP}" "cd /root/dev-scripts/ && rm -rf metallb/ && git clone ${METALLB_REPO} && mv metallb-openshift metallb && cd metallb/ && git checkout cidual"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)